### DISCUSSION TITLE
Social Links in Hugo’s Jane Theme Always Redirect to Localhost

### DISCUSSION THREAD

**Discussion by pi4630** [xianmin/hugo-theme-jane#402] [Q&A]
Hi, I’m using the Jane theme with Hugo on localhost. Despite setting external URLs in [params.social] within hugo.toml (e.g., YouTube), the social icons still redirect to localhost. Other hugo.toml changes, like the logo title, apply correctly. I couldn’t locate a social.html file or similar in the theme to troubleshoot. Is there a way to get the social links to use the specified URLs?

**Comment by xianmin** [comment]
Fixed. https://github.com/xianmin/hugo-theme-jane/commit/7f03744deb9fde9bc8c98b484b1964bc871294b6

Thanks for your feedback!
